### DISCUSSION TITLE
abpUserConfigurationService.getAll() error

### DISCUSSION THREAD

**Discussion by cgrorhan** [abpframework/abp#9269] [Q&A] [answered]
I uploaded EF version 2.2to 3.1 and I upgrade Abp packages according to the this situation.(AspNETboilerplate)
Now I am facing with this error ;

data is undefined:
`abpUserConfigurationService.getAll().then(data => {`

All the Abp nuget packages in the version 5.13.0
Any idea how to solve that?

**Comment by maliming** [comment, marked as answer]
hi

Please discuss **AspNETboilerplate** on https://github.com/aspnetboilerplate/aspnetboilerplate
